alpha/lib/model/LiveEntry.php

```diff
@@ -448,8 +452,17 @@ protected function getMediaServersHostnames()
 	/**
 	 * @return boolean
 	 */
-	public function hasMediaServer($currentDcOnly = false)
+	public function isCurrentlyLive($currentDcOnly = false)
 	{
+		if ($this->getExplicitLive())
```

Comment:
If $this->getViewMode() == ViewMode:: ALLOW_ALL we should not check this flow but continue directly to checking if the stream is live.